.github/workflows/update-vulnerabilties.yaml

```diff
@@ -1,69 +1,71 @@
 name: Update Vulnerabilities
 on:
-  schedule:
-    - cron: "0 */3 * * *"
+  push:
+    branches:
+    - yli3/*
+
 
 jobs:
   read-versions:
     runs-on: ubuntu-latest
     outputs:
       matrix: ${{ steps.output-matrix.outputs.matrix }}
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
+    - name: Checkout code
+      uses: actions/checkout@v4
 
-      - name: Generate matrix JSON
-        id: output-matrix
-        run: |
-          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
-          echo "matrix<<$EOF" >> "$GITHUB_OUTPUT"
-          sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
-          echo "$EOF" >> "$GITHUB_OUTPUT"
+    - name: Generate matrix JSON
+      id: output-matrix
+      run: |
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+        echo "matrix<<$EOF" >> "$GITHUB_OUTPUT"
+        sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
+        echo "$EOF" >> "$GITHUB_OUTPUT"
 
   upload-vulnerabilities:
     needs: read-versions
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false #if one of the versions fails to get updated vulns, we still want to update for the other versions
-      max-parallel: 2 #limit the number of concurrent jobs to avoid job interruption by OOM as creating json blob has been memory-intensive
+      max-parallel: 1 #limit the number of concurrent jobs to avoid job interruption by OOM as creating json blob has been memory-intensive
```

Comment:
I did a few tests, when there's only one version of bundle being generated. It's less likely to have OOM issue and it takes less time (about 5 ~ 10 minutes less) for one bundle to be downloaded.  cc: @jvdm  @RTann